_topic_maps/_topic_map.yml

```diff
@@ -422,7 +422,7 @@ Topics:
     File: install-sno-preparing-to-install-sno
   - Name: Installing OpenShift on a single node
     File: install-sno-installing-sno
-- Name: Installing on bare metal
+- Name: Installing on bare metal with the UPI Installer
```

Comment:
This is not correct terminology - the installation program is the same for UPI and IPI. please remove this change and line 441 as well as John Wilkins is reorganizing the bare metal content

Reply:
Incorrect, this chapter contains only information for UPI with a separate chapter for IPI:

- UPI - https://docs.openshift.com/container-platform/4.14/installing/installing_bare_metal/preparing-to-install-on-bare-metal.html
- IPI - https://docs.openshift.com/container-platform/4.14/installing/installing_bare_metal_ipi/ipi-install-overview.html

Reply:
There is no such thing as a "UPI installer" or an "IPI installer". That phrase does not make sense

Reply:
Removed